data/teamData.js

```diff
@@ -51,6 +46,16 @@ export default {
       role: "Product Manager",
       linkedin: "https://www.linkedin.com/in/alan-fang/"
     },
+    {
+      name: "Shreyas Mohan",
+      role: "Product Manager",
```

Comment:
Maybe change this to `Academy Lead`? Also do the same for Navam and move him somewhere in the lead section.